src/Tests/Microsoft.DotNet.GenAPI.Tests/SyntaxRewriterTests.cs

```diff
@@ -463,4 +463,21 @@ class B
                 """);
         }
     }
+
+    public class TypeForwardAttributeCSharpSyntaxRewriterTests : SyntaxRewriterTests
+    {
+        [Fact]
+        public void TypeForwardAttributeDeclaration()
+        {
+            CompareSyntaxTree(new TypeForwardAttributeCSharpSyntaxRewriter(),
+                original: """
+                [assembly:System.Runtime.CompilerServices.TypeForwardedToAttribute(typeof(global::System.Collections.Generic.IAsyncEnumerable<T>))]
+                [assembly:System.Runtime.CompilerServices.TypeForwardedToAttribute(typeof(global::System.Collections.Generic.IAsyncEnumerable<A, B, C>))]
+                """,
+                expected: """
+                [assembly:System.Runtime.CompilerServices.TypeForwardedToAttribute(typeof(global::System.Collections.Generic.IAsyncEnumerable<>))]
+                [assembly:System.Runtime.CompilerServices.TypeForwardedToAttribute(typeof(global::System.Collections.Generic.IAsyncEnumerable<,,>))]
```

Comment:
Nope. They are. This is not a black box test case, but just of `TypeForwardAttributeCSharpSyntaxRewriter`

Reply:
Do we also have an integration test that makes sure that the "global::" part is trimmed away?

Reply:
yes. all integration tests that we have test this. For example, if you comment the call of Simplifier, you'd get:
```C#
public global::System.Int32? Drivers { get { throw null; } }
```
instead of
```C#
public int? Drivers { get { throw null; } }
```

Reply:
But do we have integration tests for type forward attributes that make sure that the "global::" part is trimmed away?